components/dashboard/table/Filters.js

```diff
@@ -68,10 +68,10 @@ export default function Filters({
             style={{
               position: strategy,
               top: y ?? '',
-              left: x ?? '',
+              left: x>0? x??'':"14px",
             }}
           >
-            <div className="bg-white w-96 shadow-xl rounded">
+            <div className="bg-white shadow-xl rounded" style={{width: "19rem"}}>
```

Comment:
We don't use style tag to add CSS inside tags. Kindly use tailwind classes to add the width.

Reply:
Ok